### ISSUE TITLE
share trained data?

### ISSUE THREAD

**Issue by skatbr** [zeyad-mansour/lunar#185]
can anyone who have good aimbot share their trained data. You can find it in 
lunar-main\lib\data
inside there should be image's if u have trained the ai🚗

**Comment by syntax-error69** [comment]
> can anyone who have good aimbot share their trained data. You can find it in lunar-main\lib\data inside there should be image's if u have trained the ai🚗

Yeah ive got it trained on Fortnite and Apex Legends and have messed around with the settings for weeks and got the perfect settings for both and solved multiple issues that were just trial and error for days. i can send the data you got discord?

**Comment by Reddonan** [comment]
Is the aimbot working well for you

**Comment by SAOJulius** [comment]
Got some of mine about 3000 images add me on discord for them SuperJu#8495

**Comment by SAOJulius** [comment]
> 



> > Got some of mine about 3000 images add me on discord for them SuperJu#8495
> 
> Do ya mind if I get those too? My aimbot trainer data isn't the best I found out. Sent fr request

What is your discord name again might have deletet your request

**Comment by SAOJulius** [comment]
Have made a discord server for that so anyone can access the dataset. "https://discord.gg/TPpAfnHF"

**Comment by alexjohn101** [comment]
> Have made a discord server for that so anyone can access the dataset. "https://discord.gg/TPpAfnHF"

both your discord and this is showing as invalid, can you add me meciah1017#4947 thanks

**Comment by realDivineApe** [comment]
If anyone got good training files, can you send them my way? 0xStarFox#3800

**Comment by Luspsx** [comment]
> Got some of mine about 3000 images add me on discord for them SuperJu#8495

can you add me and send them too? it says the discord is invalid. my tag is lusp#0001

**Comment by torokesou** [comment]
> Have made a discord server for that so anyone can access the dataset. "https://discord.gg/TPpAfnHF"

Please new discord link

**Comment by aAtoxvvv** [comment]
> 

Hey, I just send you a fr, JordanBelford#0187

**Comment by TonttuHomo** [comment]
Plz add me too, Monnke#6804. Much appreciated!

**Comment by torokesou** [comment]
Plz add me too, torokesou#7777
